crates/sui-types/src/effects/mod.rs

```diff
@@ -348,6 +349,8 @@ pub trait TransactionEffectsAPI {
     fn events_digest(&self) -> Option<&TransactionEventsDigest>;
     fn dependencies(&self) -> &[TransactionDigest];
 
+    fn move_abort(&self) -> Option<Abort>;
```

Comment:
nit: This is in kind of a random place (order-wise) -- perhaps move it next to the `status` functions?